reference/about-reference.md

```diff
@@ -1,44 +1,38 @@
 # The reference article
 
-Reference articles provide information-oriented descriptions of specific technology parts.
-The purpose of reference articles is to describe the topic as consistently and accurately as possible, without mixing procedural or instructional content.
+Reference articles provide information-oriented descriptions of specific components or characteristics of your product.
+The purpose of a reference article is to concisely present information as a structured set of entries, with little to no procedural or instructional content.
 
-Reference articles work well when they are:
+A reference article works well when it:
 
-* Consistent in structure, language, and tone
-* Contain descriptive information that is relevant to the reference topic's overview
-* Focus on information accuracy and the facts
+* is consistent in structure, terminology, and tone.
+* contains concise, descriptive information that is relevant to its overview
+* avoids high-level usage-instructions or description for the product as a whole
 
```

Comment:
Actually the convention I have adopted (I think I got this from a talk at Write the Docs), is to use lower case when the bulleted items are completing a sentence begun in the prompt. Otherwise, each item should be capitalized. How would you feel about us doing that for our work here?

Reply:
Regarding the idea of self-documenting UIs, this is a great practice. TBH I have never written a reference page specifically for a UI, this is content that a previous contributor added in here I believe. I'm happy to delete it for now and we can address UIs in here again only if the issue comes up.

> **General comment:** Great work on this! There's a lot to like here and I have only minor wording fixes.
> 
> **Suggestion (non-blocking):** I do have one general comment about this template that I wonder if you want to work into the introduction to the About guide. My new employer has required me to read and follow the guidelines in _Developing Quality Technical Information_ by IBM. This book is about a philosophy called "progressive disclosure" that strongly advocates for making UIs self-documenting as much as possible using tooltips and other UI guidance. The goal is to cut down on the amount of reference documentation you need to write and to avoid bloated over-documentation, especially for reference material where the function of fields are fairly obvious. I don't know if you want to mention that as a caveat in this template, e.g. decisions about when to use it and when not to use it?
> 
> If this is out of scope for your PR and you don't want to deal with that, we can just spin my idea off as a separate issue. Just a suggestion.

Reply:
I'll defer to you on this issue. Consider this matter resolved.